docs/02-deployment-guide.md

```diff
@@ -122,6 +124,13 @@ Parameters after the environment ID will be passed to a `cdk destroy` command.
 
 ## Deployment
 
+### Install Prerequisite Software
+
+To run deployment scripts you need to have the prerequesite software installed in your environment. The setup section
+in the [dev guide](11-dev-guide.md#install-prerequisite-software) describes two easy ways to get an environment setup
+quickly. The following commands assume that you are in an environment that has the prerequesite software installed and
+you are starting in the root of the repository.
```

Comment:
This reads like it should be part of the "Get your environment set up" section. That does refer to the getting started guide, which refers to the dev guide for building the system. If the getting started guide doesn't explain the setup well enough, maybe we should either improve the getting started guide, or add some extra information next to where it refers you to the getting started guide?